ddtrace/tracer.py

```diff
@@ -632,6 +663,7 @@ def set_service_info(self, *args, **kwargs):
         return
 
     def wrap(self, name=None, service=None, resource=None, span_type=None):
+        # type: (Optional[str], Optional[str], Optional[str], Optional[str]) -> Callable[[Callable[..., Any]], Callable[..., Any]]  # noqa
```

Comment:
"noqa" because the line is too long? are we better following this pattern we use as well?

```suggestion
    def wrap(
        self,
        name=None,  # type: Optional[str]
        service=None,  # type: Optional[str]
        resource=None,  # type: Optional[str]
        span_type=None,  # type: Optional[str]
    ):
        # type: (...) -> Callable[[Callable[..., Any]], Callable[..., Any]]
```

Reply:
Completely didn't occur to me I can use both type hinting styles haha, fixed